apps/ui/.env.schema

```diff
@@ -1 +1,2 @@
-VITE_APPLICATION_API_URL=http://localhost:3000
+API_URL=http://localhost:3001
+PORT=3000
```

Comment:
`/apps/api/.env.schema` needs a similar update; API is set to use port 3000 so the two apps will need to run on different ports

Reply:
Simplified my changes, I no longer fix the UI port in development. This env variable is removed, all ports for local dev will appear the same as they were before.